en/admin/config.rst

```diff
@@ -1992,6 +2008,10 @@ The following are other parameters. The type and value range for each parameter
 
     **optimizer_enable_merge_join** is a parameter to specify whether to include sort merge join plan as a candidate of query plans or not. The default is **no**. Regarding sort merge join, see :ref:`sql-hint`.
 
+**use_stat_estimation**
+
+    **use_stat_estimation** is a parameter to specify whether to use the estimated information in calculating statistics or not. The default is no. The estimated information generated by the heap manager during DML is related to the number of added objects. it is relatively accurate for number of total objects, NOT for number of distinct values.
+
```

Comment:
during DML --> while processing DML
related to the number of added objects --> associated with the number of added objects
it is relatively accurate for number of total objects, NOT for number of distinct values.
-->
It is relatively accurate for the number of total objects, not for the number of distinct values.

Reply:
I fixed it